py-polars/polars/internals/frame.py

```diff
@@ -880,7 +890,14 @@ def transpose(
     def to_parquet(
         self,
         file: Union[str, Path, BytesIO],
-        compression: Optional[str] = "snappy",
+        compression: Optional[
+            Union[
+                Literal[
```

Comment:
Cool! I didn't know we could use this in typing.

Reply:
Yes, we can. The only annoying thing with literal is that it only can typecheck:
```python
df.to_ipc(..., compression="uncompressed")

# or
compression: Literal["uncompressed"] = "uncompressed"
df.to_ipc(..., compresssion=compression)
```
But not when the string is in a variable:
```python
compression="uncompressed"
df.to_ipc(..., compresssion=compression)
```